### PR TITLE
First implementation for OrderPlaced app components

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?

This pull request adds the first iteration of each component that will be part of OrderPlaced app.

#### What problem is this solving?

Provides initial layout and implementation of each component.

#### How should this be manually tested?
Run `vtex link` on repository root and follow the instructions provided by Renderer.

#### Types of changes
- [  ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [  ] Breaking change (fix or feature that would cause existing functionality to change)
- [  ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Header/Header.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          <MailIcon />
+          <span className="pl4">REENVIAR E-MAIL</span>
+        </p>
+      </div>
+
+      {/* Order important info */}
+      <Warnings />
+
+      {/* Order Summary (not always displayed) */}
+      {showSummary ? (<OrderSummary />) : <void />}
```
Comment (klzns) [inline react/components/Header/Header.js]: Nao conhecia esse void, algum motivo pra não usar `null`?

File: react/components/OrderInfo/BasicInfo.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { Button, Alert, IconCaretDown } from 'vtex.styleguide'
+
+const BasicInfo = () => {
+  const twoOrders = true
+
+  return (
+    <div className="flex flex-column">
+      <div className="order-number-date">
+        <p className="f4">
```
Comment (klzns) [inline react/components/OrderInfo/BasicInfo.js]: Usar typography tokens ao inves de `f4`
https://styleguide.vtex.com/#/Styles?id=typography

Comment (victorhmp) [reply react/components/OrderInfo/BasicInfo.js]: Beleza! Vou trocar e usar de agora em diante, não tinha visto essa parte do Style Guide ainda.

Comment (klzns) [reply react/components/OrderInfo/BasicInfo.js]: Publicamos hj hehe

File: react/components/Header/OrderStatus.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
```
Comment (salesfelipe) [inline react/components/Header/OrderStatus.js]: remove emptyline

File: react/components/Header/OrderSummary.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { Box } from 'vtex.styleguide'
+
+import DeliveryIcon from '../Icons/DeliveryIcon'
+import PickUpIcon from '../Icons/PickUpIcon'
+
+const OrderSummary = () => {
+  return (
+    <div>
```
Comment (salesfelipe) [inline react/components/Header/OrderSummary.js]: Is this div really necessary?

File: react/components/Header/Warnings.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { IconWarning } from 'vtex.styleguide'
+
+const Info = () => {
+  const twoOrders = true
+  return (
+    <div>
```
Comment (salesfelipe) [inline react/components/Header/Warnings.js]: remove unnecessary div

**Pull request merged**